op-node/service.go

```diff
@@ -289,9 +289,19 @@ func NewSyncConfig(ctx *cli.Context, log log.Logger) (*sync.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	//fastnode config
+	elTriggerGap := ctx.Int(flags.ELTriggerGap.Name)
+	if ctx.Bool(flags.FastnodeMode.Name) {
```

Comment:
Since we have introduced this flag, I think we should set `--allow-insecure-no-tries` automatically when we enable this flag.

Reply:
These two configurations are in op-node and op-geth respectively, and it seems that achieving this linkage is not very easy.